pages/docs/reference/js-overview.md

```diff
@@ -5,18 +5,22 @@ category: "Introduction"
 title: "Kotlin for JavaScript"
 ---
 
-# Kotlin JavaScript Overview
+# Kotlin/JS Overview
 
-Kotlin provides the ability to target JavaScript. It does so by transpiling Kotlin to JavaScript. The current implementation targets ECMAScript 5.1 but there are plans to eventually target ECMAScript 2015 as well.
+Kotlin provides the ability to target JavaScript. It does so by transpiling Kotlin to JavaScript. The current implementation of Kotlin/JS targets [ES5](https://www.ecma-international.org/ecma-262/5.1/). Support for targeting [ES6](http://www.ecma-international.org/ecma-262/6.0/) directly is currently in development.
```

Comment:
Thanks for bringing my attention to this. After some consideration, I've restructured and rewritten major parts of this page. It now features the multiplatform story more prominently, and moves technical details a bit further down. This will hopefully help people get a better idea of the technology and its use cases "at a glance".

I've removed references to ES6.

Kindly re-review the `js-overview` page, since the page has received a complete overhaul. :)

Reply:
I like the new version of the JS overview page. It's good that you've covered use cases in detail. Do you have sample projects you could refer to? It would be great not just to describe a use case but provide a _real_ use case in a project.

Reply:
Besides the topics covered in the hands-ons, I don't have anything ready to link. We can add some case studies (and consolidated examples) at a later point to this page though, I agree.